lib/zoom/actions/roles.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module Zoom
+  module Actions
+    module Roles
+      def roles_list(*args)
+        options = Params.new(Utils.extract_options!(args))
+        Utils.parse_response self.class.get("/roles", query: options, headers: request_headers)
+      end
+
+      def roles_create(*args)
+        options = Params.new(Utils.extract_options!(args))
+        options.require(%i[name]).permit(%i[description privileges])
```

Comment:
Minor style nit-pick but only use literal syntax when  listing two or more symbols
```suggestion
        options.require(:name).permit(%i[description privileges])
```